### ISSUE TITLE
Add Cashdrop Currency Identification

### ISSUE THREAD

**Issue by balqaasem** [setheum/setheum#207]
We can just use `StableCurrencyIds` and say that -
```
let containing = T::StableCurrencyIds.contains(transacted_currency_id):
if !containing {
	Self::cashdrop_setter();
} else {
	Self::cashdrop_settcurrency(transacted_currency_id);
}
```

**Comment by balqaasem** [comment]
works on continuig #179

**Comment by balqaasem** [comment]
as a part of #164

**Comment by balqaasem** [comment]
Done! Closes here in Serp settpay #383
